### ISSUE TITLE
Not working when module.name_mapper is used

### ISSUE THREAD

**Issue by alexeychikk** [brigand/babel-plugin-flow-react-proptypes#198]
Hi. I'm trying to import type with alias:
```
module.name_mapper='^@app\/\(.*\)$' -> '<PROJECT_ROOT>/src/\1'
```
Import looks like:
```js
import type { Value } from '@app/flow-typed/slate';

type PostEditorProps = {
  content: Value,
  updateEditorContent: Function
};
```
No PropTypes errors in console, however there is an error when I import it with:
```js
import type { Value } from '../../../../flow-typed/slate';
```
Webpack aliases are configured properly for sure because they are used and work ok.

**Comment by brigand** [comment]
See https://github.com/brigand/babel-plugin-flow-react-proptypes/issues/197 https://github.com/brigand/babel-plugin-flow-react-proptypes/pull/186

TL;DR we need to add some form of config option to allow this, but I'm not sure on the details of how it should work.
